packages/starter/example/components/ContextProvider.tsx

```diff
@@ -1,22 +1,24 @@
-import { createTheme, StyledEngineProvider, ThemeProvider } from '@mui/material';
-import { deepPurple, pink } from '@mui/material/colors';
-import { WalletModalProvider as AntDesignWalletModalProvider } from '@solana/wallet-adapter-ant-design';
-import { WalletAdapterNetwork, WalletError } from '@solana/wallet-adapter-base';
-import { WalletDialogProvider as MaterialUIWalletDialogProvider } from '@solana/wallet-adapter-material-ui';
+import { AutoConnectProvider, useAutoConnect } from './AutoConnectProvider';
```

Comment:
Please limit changes to adding the package only. We don't add all new wallets to the example or starter projects.

Reply:
It doesn't look like this was fixed.